pfinance/functions.py

```diff
@@ -85,7 +117,7 @@ def loan_payment(principal: float, interest_rate: float, payment_frequency: int,
     return loan_amount * effective_rate * (1 + effective_rate) ** term / ((1 + effective_rate) ** term - 1)
 
 
-def discounted_cash_flow(cash_flows: list[float], discount_rate: float):
+def discounted_cash_flow(cash_flows: list[float], discount_rate: float) -> float:
```

Comment:
Can you rebase this branch onto master? I think these changes already exist there and I'm not sure why they still show up here